components/questionCount.js

```diff
@@ -1,10 +1,137 @@
 import React from "react";
+import {useState} from "react";
+import {Button} from "react-bootstrap";
 import PropTypes from "prop-types";
+import {isMobile} from "react-device-detect";
+import ArrowUpwardRoundedIcon from "@material-ui/icons/ArrowUpward";
+import ArrowDownwardRoundedIcon from "@material-ui/icons/ArrowDownwardRounded";
+import ArrowForwardRoundedIcon from "@material-ui/icons/ArrowForwardRounded";
+import ArrowBackRoundedIcon from "@material-ui/icons/ArrowBackRounded";
+import Modal from "react-bootstrap/Modal";
 
 function QuestionCount(props) {
+  const [show, setShow] = useState(false);
+  const [displayUnsure] = useState(props.display);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
   return (
     <h2 className="questionCount">
       <span>{props.counter + 1}</span>/<span>{props.total}</span>
+      {!isMobile && (
+        <span className="col-sm-12 text-right" style={{marginLeft: "14rem"}}>
+          <button
+            type="button"
+            onClick={handleShow}
+            className="btn btn-warning"
+            style={{
+              color: "white",
+              fontSize: "14px",
+              backgroundColor: "#18A0FB",
+              borderWidth: 0,
+              marginBottom: 2,
+              marginTop: 0,
+            }}
+          >
+            Keyboard Shortcuts
+          </button>
+        </span>
+      )}
+      <Modal size="sm" show={show} onHide={handleClose}>
```

Comment:
You could include the Modal in the `!isMobile` if clause, so that it does not even get rendered on mobile vs being rendered but hidden. If you need an overarching block to include both the `<span>` and the `<Modal>` without adding another block, you can enclose both in a `<>  </>` block

Reply:
Resolved in commit 76ff221